lib/mailtrap/batch_sender.rb

```
@@ -0,0 +1,58 @@
+          next unless request[field].is_a?(Array)
+    
+          request[field].compact.each do |recipient|
+            begin
+              Mailtrap::Validators::EmailValidator.validate!(recipient[:email], field_name: "#{field}[:email] in request ##{index + 1}")
+            rescue ArgumentError => e
+              raise ArgumentError, e.message
+            end
+          end
+        end
+      end
+    end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `validate_requests!` is too high. [<5, 17, 9> 19.87/17]

(Metrics/AbcSize)